adcm_aio_client/objects/rbac/_types.py

```diff
@@ -0,0 +1,61 @@
+from enum import Enum
+from typing import TYPE_CHECKING, Annotated, NotRequired, Self, TypedDict
+
+from pydantic import BaseModel, ConfigDict, Field
+
+from adcm_aio_client.requesters import DefaultRequester
+
+if TYPE_CHECKING:
+    from adcm_aio_client.objects.rbac._user import LocalUser
+
+
+class UserStatus(str, Enum):
+    ACTIVE = "active"
+    INACTIVE = "inactive"
+
+
+class SourceType(str, Enum):
+    LOCAL = "local"
+    LDAP = "ldap"
+
+
+class LocalUserData(BaseModel):
+    id: Annotated[int | None, Field(default=None, gt=0)]
+    username: Annotated[str | None, Field(default=None)]
+    password: Annotated[str | None, Field(default=None)]
+    is_super_user: Annotated[bool | None, Field(default=None, serialization_alias="isSuperUser")]
+    first_name: Annotated[str | None, Field(default=None, serialization_alias="firstName")]
+    last_name: Annotated[str | None, Field(default=None, serialization_alias="lastName")]
+    email: Annotated[str | None, Field(default=None)]
+
+    model_config = ConfigDict(extra="forbid", arbitrary_types_allowed=True)
+
+
+class LocalUserLazy(LocalUserData):
```

Comment:
Maybe `LocalUserData` should be in types, but `LocalUserLazy` in `_user`... does it makes sense?